etna/config.yml.template

```diff
@@ -0,0 +1,16 @@
+---
```

Comment:
Since these just point to the hosts, in this case it might be okay to include the `config.yml` in the repo? I guess it depends on how we do the packaging for installation.

Reply:
Seems like a bad habit. I would like to see these configs become database records soon in any case.

Reply:
Then it seems like installing with docker and a db for clients would be the way to go...

Reply:
Hmm.  I disagree.  What advantages are there to configuring hosts in a database?
While I do believe many parts of our config should be in a db, I think hostnames should be static, since it is generally not possible to dynamically update hostnames without breakage (we don't control clients, we'd be forced to ask every client user to update their ... database?)

Reply:
Also, in general, any state we keep for clients I think should just be in flat files.  If we need to address a user issue, asking them to edit a file is a faaaar more friendly interface for them than asking them to edit a local db and learn sql.  config.yml works great because _we already have the tooling and code_ for loading this file when running commands. 

That's my thoughts in general.

Reply:
I don't disagree that some of the config.yml stuff could become db records though.  Hosts I think are different, and in general, I think it makes sense for client gems to have a config.yml themselves.

Reply:
>I don't disagree that some of the config.yml stuff could become db records though. Hosts I think are different, and in general, I think it makes sense for client gems to have a config.yml themselves.

I agree with this, the previous comment was just shower talk. I'd still avoid checking in the actual config.yml however. My rule of thumb is if "ucsf.edu" is in some string, it should not be checked in.

Reply:
That, is a good point.
Perhaps we need some process to 'build' the config.yml that is provided to clients.